lib/dfe/reference_data/itt/cycles.rb

```diff
@@ -108,19 +108,19 @@ def self.make_local_time(*args)
             apply_1_deadline: nil, # This deadline is not applicable for this cycle
             apply_2_deadline: make_local_time(2025, 9, 16, 18), # 3rd Tuesday in September. This is the final and only deadline to send ITT applications in the cycle
             provider_decision_deadline: make_local_time(2025, 9, 24, 23, 59, 59), # 1 week and a day after Apply 2 deadline
-            find_closes: make_local_time(2025, 10, 6, 23, 59, 59), # The evening before find opens in the new cycle
+            find_closes: make_local_time(2025, 9, 30, 23, 59, 59), # The evening before find opens in the new cycle
             non_working_days: {
               christmas: Date.new(2024, 12, 16)..Date.new(2025, 1, 3),
               easter: Date.new(2025, 4, 7)..Date.new(2025, 4, 21)
             }
           },
           '2025-2026' => {
```

Comment:
Please can you add a comment to show the link to the gov.uk website for reference on the decision made: https://www.gov.uk/government/publications/recruiting-postgraduate-trainee-teachers-recruitment-cycle-dates/recruiting-postgraduate-trainee-teachers-recruitment-cycle-dates